src/component/svg.js

```diff
@@ -0,0 +1,16 @@
+import SvgJS, {
+  Path as PathJS,
+  Circle as CircleJS,
+  Defs as DefsJS,
+  Stop as StopJS,
+  LinearGradient as LinearGradientJS,
+} from 'svgs';
+
+export const Path = PathJS;
+export const Circle = CircleJS;
+export const Defs = DefsJS;
+export const Stop = StopJS;
+export const LinearGradient = LinearGradientJS;
+
+const Svg = SvgJS;
+export default Svg;
```

Comment:
We should re-export all here. Try

export * from 'svgs';

See https://stackoverflow.com/questions/29844074/es6-export-all-values-from-object